io/net/virt-net/network_virtualization.py

```diff
@@ -382,6 +378,12 @@ def test_rmdev_viosfailover(self):
         '''
         using mrdev and mkdev command to check vios failover works
         '''
+
+        self.session = Session(self.vios_ip, user=self.vios_user,
+                               password=self.vios_pwd)
+        if not self.session.connect():
+                self.fail("Failed connecting to VIOS")
```

Comment:
@cforno12 this check is needed but don't kept inside the test, keep in setUp() it self.

Reply:
@smruti77 I asked for this to be moved inside test since there was only one test using it.

Reply:
@narasimhan-v  ok fine, i didn't saw that comments.